doc/filters/spaceless.rst

```diff
@@ -1,10 +1,6 @@
 ``spaceless``
 =============
 
-.. versionadded:: 1.38
```

Comment:
this should actually say `2.7.0`

Reply:
I created a PR: https://github.com/twigphp/Twig/pull/3436